### ISSUE TITLE
'skip navigation' button should become visible when it has focus

### ISSUE THREAD

**Issue by moloko** [adaptlearning/adapt_framework#3040]
At least, according to [the page about this accessibility feature in Techniques for WCAG 2.0](https://www.w3.org/TR/WCAG20-TECHS/G1.html):
> Check that the link is either always visible or visible when it has keyboard focus

I'm sure at one point it did display when it had focus but this functionality seems to have been lost/changed at some point.

**Comment by moloko** [comment]
Also just to note it has a class `a11y-ignore-focus` attached to it but I can only find one other reference to this, here:
https://github.com/adaptlearning/adapt_framework/blob/5a3de6952f016e70aacd986a9e6c71e0ebed462b/src/core/js/helpers.js#L326-L330

Is this something 'legacy' that needs to be removed?
